entrypoint.sh

```diff
@@ -36,7 +36,10 @@ fi
 # Setup game server
 if [ ! -f "${GAMESERVER}" ]; then
     echo "creating ./${GAMESERVER}"
-   ./linuxgsm.sh ${GAMESERVER}
+    exec_name=$(./linuxgsm.sh ${GAMESERVER} | grep -o ${GAMESERVER}-. | head -1)
+    if [ ! -z "$exec_name" ]; then
+        export GAMESERVER=$exec_name
+    fi
```

Comment:
I think a better way of doing this would be using a [bash parameter expansion substring](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html)

[A REPL for this is available here](https://replit.com/@Neo-Desktop/bash-parameter-expansion-substring#main.sh)



```suggestion
   ./linuxgsm.sh ${GAMESERVER%-*}
```

Reply:
you may need to alter other parts of the script as well (such as `exit_handler()`)